packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -102,6 +102,57 @@ class KnexAdapter extends BaseKeystoneAdapter {
     return fkResult;
   }
 
+  async _createAdjacencyTable({ tableName, relationshipFa, leftListAdapter }) {
+    // Create an adjacency table for the (many to many) relationship field adapter provided
+    const dbAdapter = this;
+    try {
+      console.log(`Dropping table ${tableName}`);
+      await dbAdapter.schema().dropTableIfExists(tableName);
```

Comment:
Do you don't want to drop the table every time? Won't this loose some relationships?

Reply:
If we're at this point of the code it's assumed that we're recreating the database, so we cleanup the table we're about to create just to be 100% certain we're starting from a clean slate.